src/main/java/leduc/command/FindCommand.java

```diff
@@ -115,9 +115,9 @@ public void execute(TaskList tasks, Ui ui, Storage storage) throws EmptyArgument
 
         }
         if (result.isEmpty()) {
-            ui.showFindNotMatching();
+            ui.display("\t There is no matching tasks in your list");
```

Comment:
You should try to use ui.showFindNotMatching(), and ui.showFindMatching(result) to display your result